task/kafka-publish/src/main/java/com/netflix/conductor/contribs/tasks/kafka/KafkaPublishTask.java

```diff
@@ -102,8 +102,15 @@ public void start(WorkflowModel workflow, TaskModel task, WorkflowExecutor execu
 
         Optional<String> traceId = span.getTraceId();
         if (traceId.isPresent()) {
+            span.setUserIdentifier(workflow.getUserIdentifier());
+
             Map<String, Object> headers = new HashMap<>();
             headers.put(tracingProvider.getTraceHeader(), span.getTraceId().get());
+
+            if (workflow.getUserIdentifier() != null) {
+                headers.put("user-uuid", workflow.getUserIdentifier());
```

Comment:
we are using the header in different places as `X-ROUTIFIC-USER`, for example [here](https://github.com/routific/map-service/blob/00500d0f815a61e6497e3b8ee6ad7cf432ec1816/src/routes.ts#L15), do you think it makes sense to keep it like this?